test/test_jit.py

```diff
@@ -13796,6 +13796,15 @@ def test_non_primitive_types(x):
         out = test_non_primitive_types(_MyNamedTuple(value=torch.tensor(5.0)))
         self.assertEqual(out, torch.tensor(6.0))
 
+        def test_error():
```

Comment:
I think this deserves to be its own test case rather than being part of `test_isinstance_metacompile`

Reply:
got it